src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/resource/PrisonApiResource.kt

```diff
@@ -15,13 +15,13 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.config.ErrorResponse
-import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.dto.prisonapi.NonAssociationDetails
+import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.dto.prisonapi.LegacyNonAssociationDetails
 import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.service.NonAssociationsService
 
 @RestController
 @Validated
 @RequestMapping("/legacy/api", produces = [MediaType.APPLICATION_JSON_VALUE])
-@Tag(name = "Non-Associations", description = "Retrieve non association details from prison-api")
+@Tag(name = "Legacy non-associations-details", description = "Retrieve non association details from Prison API")
```

Comment:
should we remove `@PreAuthorize("hasRole('ROLE_NON_ASSOCIATIONS')")` from this, the old api because it's not how prison-api did permissions?

Reply:
Yeah good call, I'll do 👍

Reply:
Removed that role requirement for the legacy endpoint: https://github.com/ministryofjustice/hmpps-non-associations-api/pull/22/commits/07ea92464af5ad98b6b9de39bd737cad4e473cb9